src/gtimelog/timelog.py

```diff
@@ -283,6 +289,9 @@ def grouped_entries(self, skip_first=True):
         work = {}
         slack = {}
         for start, stop, duration, tags, entry in self.all_entries():
+            cat, task = self.split_category(entry)
+            entry = task or ''
+            entry = cat and cat + ': ' + entry or entry
```

Comment:
I think `task` is guaranteed to be a string, so the `or ''` shouldn't do anything.

I would prefer the ternary `if` expression instead of the `x and a or b` hack.

Reply:
Task may also be None. See `split_category()`

Reply:
> I would prefer the ternary if expression instead of the x and a or b hack

If you prefer, I can change this. Just want to point out that `x and a or b` is not a hack but perfect valid and actually nice python syntax ;-)